docs/about-mono/releases/4.6.0.md

```diff
@@ -59,6 +59,44 @@ For custom implementations, the type should subclass the
 `Mono.Security.Interface.MonoTlsProvider` interface from the
 `Mono.Security` assembly and have a public parameterless constructor.
 
+### Mono symbolicate
+
+The previous version of mono-symbolicate was not created with the multiple symbols versions in mind.
```

Comment:
what's the "multiple symbols version", do we have that documented somewhere? please add a link or explain here.